eclair-core/src/main/scala/fr/acinq/eclair/api/NewService.scala

```diff
@@ -0,0 +1,251 @@
+                    case (invoice, Some(overrideAmount)) => complete(eclairApi.findRoute(invoice.nodeId, overrideAmount, invoice.routingInfo))
+                    case _ => reject(MalformedFormFieldRejection("invoice", "The invoice must have an amount or you need to specify one using 'amountMsat'"))
+                  }
+                } ~ path("findRouteByNode") {
+                formFields("nodeId".as[PublicKey], "amountMsat".as[Long]) { (nodeId, amount) =>
+                  complete(eclairApi.findRoute(nodeId, amount))
+                }
+              } ~
+                path("sendToInvoice") {
+                  formFields("invoice".as[PaymentRequest], "amountMsat".as[Long].?) {
+                    case (invoice@PaymentRequest(_, Some(amount), _, nodeId, _, _), None) =>
+                      complete(eclairApi.send(nodeId, amount.toLong, invoice.paymentHash, invoice.routingInfo, invoice.minFinalCltvExpiry))
+                    case (invoice, Some(overrideAmount)) =>
+                      complete(eclairApi.send(invoice.nodeId, overrideAmount, invoice.paymentHash, invoice.routingInfo, invoice.minFinalCltvExpiry))
+                    case _ => reject(MalformedFormFieldRejection("invoice", "The invoice must have an amount or you need to specify one using the field 'amountMsat'"))
+                  }
+                } ~
+                path("sendToNode") {
+                  formFields("amountMsat".as[Long], "paymentHash".as[ByteVector32](sha256HashUnmarshaller), "nodeId".as[PublicKey]) { (amountMsat, paymentHash, nodeId) =>
+                    complete(eclairApi.send(nodeId, amountMsat, paymentHash))
+                  }
+                } ~
+                path("checkpayment") {
+                  formFields("paymentHash".as[ByteVector32](sha256HashUnmarshaller)) { paymentHash =>
+                    complete(eclairApi.checkpayment(paymentHash))
+                  } ~ formFields("invoice".as[PaymentRequest]) { invoice =>
+                    complete(eclairApi.checkpayment(invoice.paymentHash))
+                  }
+                } ~
+                path("audit") {
+                  formFields("from".as[Long].?, "to".as[Long].?) { (from, to) =>
+                    complete(eclairApi.audit(from, to))
+                  }
+                } ~
+                path("networkfees") {
+                  formFields("from".as[Long].?, "to".as[Long].?) { (from, to) =>
+                    complete(eclairApi.networkFees(from, to))
+                  }
+                } ~
+                path("channelstats") {
+                  complete(eclairApi.channelStats())
+                } ~
+                path("ws") {
+                  handleWebSocketMessages(makeSocketHandler)
+                }
+            }
+          }
+        }
+      }
+    }
+  }
+
+  val help = List(
```

Comment:
Let's move this to `eclair-cli` as mentioned above.

Reply:
Done in 7f63104